### PR TITLE
Adds tablet sizing to footer

### PR DESCRIPTION
**Pull request by ljwedin**
Changes wrapping in the HTML to make arranging the components for the different sizes possible. Adds styling for tablet size.

### CODE REVIEW

**Review by smaristeinar — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -132,10 +132,13 @@
     </div>
 
     <footer>
-        <div>
-            <img class="socialMediaIcon" src="assets/instagram.svg">
-            <img class="socialMediaIcon" src="assets/facebook.svg">
-            <img class="socialMediaIcon" src="assets/linkedin.svg">
+        <div class="logoSoMeWrapper">
+            <img class="footerLogoSoMe" src="assets/logoDarkBlue.svg" />
```
Comment (postmodernistx) [inline index.html:136]: There should be alt text for accessibility here.

Consider WebAIM's [guidelines](https://webaim.org/techniques/alttext/).

File: index.html [hunk 2]
```diff
@@ -132,10 +132,13 @@
...
+            <div>
+                <img class="socialMediaIcon" src="assets/instagram.svg" />
```
Comment (postmodernistx) [inline index.html:138]: These need alt text too.

File: style.css.map [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (postmodernistx) [inline style.css.map:1]: I don't think the map file needs to be committed for version control :) But it's OK to have it.

**Review by postmodernistx — APPROVED** [review]
Looks good!

**Pull request merged**